docker-compose.yml

```
@@ -0,0 +1,44 @@
+version: "3.7"
+services:
+  db:
+    image: postgres:12
+    environment:
+      - POSTGRES_DB=${DATABASE_NAME:-navody_slovensko_digital_development}
+      - POSTGRES_HOST_AUTH_METHOD=trust
+    expose:
+      - 5432/tcp
+    networks:
+      - navody
```

pouzil by som cely nazov (ak nam network vobec treba)

```suggestion
      - navody_slovensko_digital
```

---

V podstate nam network ani netreba, suhlasim s odstranenim, bude to strucnejsie :+1:  
(takto sa bude network volat `navodydigital_default` - teda v zavislosti, ako ma kto pomenovany root git directory).